website/identifiers/clients/datacite.py

```diff
@@ -14,14 +14,12 @@
 
 class DataCiteClient(AbstractIdentifierClient):
 
-    def __init__(self, base_url, prefix, client=None):
-        self.base_url = base_url
-        self.prefix = prefix
-        self._client = client or DataCiteMDSClient(
-            url=self.base_url,
+    def __init__(self, node):
```

Comment:
this is a good change! thank you!

Is there any chance that the DATACITE_URL and/or PREFIX won't be set? If so, I would raise an exception so that `get_doi_client` can return `None` as it was before in that case